blazor/maps/polygon.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: post
+title: Polygon shape in Blazor Maps Component | Syncfusion
+description: Checkout and learn here all about polygon shape in Syncfusion Blazor Maps component and much more details.
+platform: Blazor
+control: Maps
+documentation: ug
+---
+
+# Polygon shape in Blazor Maps Component
+
+The Polygon shapes can be rendered over the geometry map or an online map using the [Points](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Maps.MapsPolygon.html#Syncfusion_Blazor_Maps_MapsPolygon_Points) property in the [MapsPolygon](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Maps.MapsPolygon.html) tag in the Maps component.
```

Comment:
The Maps component allows users to add polygon shapes to a geometry map or an online map by using the properties in the [MapsPolygon](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Maps.MapsPolygon.html) tag. This section describes how to add polygon shapes to the map and how to customize them.